### ISSUE TITLE
Hello, could we write your SnakeGame together?

### ISSUE THREAD

**Issue by danon** [KevinMendieta/SnakeGame#1]
:) Me and my friend could write this game with you

**Comment by KevinMendietaP** [comment]
Yes, of course, I'm actually beginner on crating games on Java. I know more about applications and bit of web pages.

**Comment by danon** [comment]
Ok, we should chat on Hangouts about our workflow. 
How can we share this file? Like, you change one thing, and I change one thing? Or I change one file and you change other file? Or as soon as we write something, we commit and push so the other person can see changes?

I want to avoid merge conflicts:)

**Comment by KevinMendietaP** [comment]
Oh well i dont know, what do you think about handling two branches of the game?
You can found me like xdkevinxd12@gma ...

**Comment by danon** [comment]
We would still have to merge them anyway, so this would only put the problem further in time. 

Only reasonable way is:
a) not edit the same file at once 
b)  if we must edit same file, commit and push as fast as possible, to avoid the other person working on old file.

Is this OK?

**Comment by KevinMendietaP** [comment]
Yeah no problem it's ok for me.
So actually the bases of the game are done. i haven't comment so much, so i'm gonna try to comment and push the results.
I will stay away for a time because of the holidays so i dont know when i will be back.

**Comment by danon** [comment]
Don't comment anything :) I can read the code just fine! 

PS; could you change from NetBeans to IntelliJ Idea? If this is not a problem? I really really really love the AutoFormat code there (just click Ctrl+Alt+L) and NetBeans doesn't have it.

**Comment by KevinMendietaP** [comment]
Yeah no problem it's good to know more editors :p
